scripts.js

```diff
@@ -39,6 +42,23 @@ getJSON('https://raw.githubusercontent.com/openfootball/football.json/master/202
     eplMatchlist = data;
 })
 
+function getScoreColor(score1, score2) {
+
+    switch (score1) {
+        case score1 > score2:
+            return 'red';
+
+        case score1 < score2:
+            return 'green';
+
+        case score1 == score2:
+            return 'grey';
+
+        default:
+            return null;
+    }
+}
+
 $(document).ready(function () {
     score_card_prefab = document.getElementById('card_start').cloneNode(true);
 
```

Comment:
function too long need to break it up into smaller pieces

- set logos
- set text
- set scores